azure-pipelines.yml

```diff
@@ -401,14 +401,20 @@ stages:
         steps:
           - bash: |
               DOCKER_TAG="$(python.version)_$(IMAGE_SUFFIX)"
-              PYTEST="docker run --network=host greatexpectations/test:${DOCKER_TAG} bash -c \""
+              PYTEST="docker run -e AWS_ACCESS_KEY_ID=$(AWS_ACCESS_KEY_ID) "
+              PYTEST+="-e AWS_SECRET_ACCESS_KEY=$(AWS_SECRET_ACCESS_KEY) -e AWS_DEFAULT_REGION=$(AWS_DEFAULT_REGION) "
+              PYTEST+="--network=host greatexpectations/test:${DOCKER_TAG} bash -c \""
```

Comment:
Not relevant to your PR but instead of creating one long string command and using `eval`, we can probably use `bash -x <script>`. This LGTM though